log/log.go

```diff
@@ -85,6 +107,16 @@ func NewLogger(cfg Config) (*zap.SugaredLogger, *zap.AtomicLevel, error) {
 	return withOptions.Sugar(), &level, nil
 }
 
+// NewTestLogger returns a new logger for testing purposes and the logs it observed.
+func NewTestLogger(cfg Config) (*zap.SugaredLogger, *observer.ObservedLogs, error) {
+	level, err := zapcore.ParseLevel(cfg.Level)
+	if err != nil {
+		return nil, nil, err
+	}
+	observedLogger, observedLogs := observer.New(level)
+	return zap.New(observedLogger).Sugar(), observedLogs, nil
+}
+
```

Comment:
same here. Check Init logs in other test

Reply:
@ARR552 see my comment above.